metricshub-doc/src/site/markdown/install.md

```diff
@@ -173,9 +182,9 @@ c:\Program Files\MetricsHub> MetricsHubServiceManager
 
 > Note: Run `CMD.EXE` or `Windows Terminal` with elevated privileges (**Run As Administrator**).
 
-This will start **${solutionName}** with the default **MetricsHub Service**'s configuration file: `C:\ProgramData\MetricsHub\config\metricshub.yaml`.
+This will start **${solutionName}** with the default configuration file `C:\ProgramData\MetricsHub\config\metricshub.yaml`.
 
-You can start **${solutionName}** with an alternate **MetricsHub Service**'s configuration file with the command below:
+Run the command below to can start **${solutionName}** with an alternate configuration file:

```

Comment:
`Run the command below to start **${solutionName}** with an alternate configuration file:`